server/src/db.js

```diff
@@ -1,11 +1,8 @@
-let user = process.env.DB_USER || process.env.USER;
-let pass = process.env.DB_PASS;
-let host = process.env.DB_HOST || "localhost:5432";
-
-pass = pass ? ":" + pass : "";
+const config = require("./config").root();
 
 let pg = require("pg");
-let constr = `postgres://${user}${pass}@${host}/${user}`;
+let credentials = config.db.password ? `${config.db.user}:${config.db.password}` : config.db.user;
```

Comment:
I think it's convenient to have separate params—certainly for the username and password—that I can override with separate environment variables, instead of building the full URL myself. But I can change it to the URL if you'd like.